lib/opensusebasetest.pm

```diff
@@ -29,6 +30,53 @@ sub export_logs {
     select_console 'root-console';
     save_screenshot;
 
+    # check if wicked is installed and turn on debug, get all needed wicked logs if network interface is down
+    if (script_run 'rpm -q wicked') {
```

Comment:
no I mean just inside the export_logs function call the code I wrote. So before L27 add
```perl
sub export_wicked_logs {
…
}
```
and after L31 add
```perl
my $wicked_installed = script_run 'rpm -q wicked';
export_wicked_logs if $wicked_installed;
```
so a refactoring of what you proposed, no change in behaviour.